docs/gambit/mutation-verifier.md

```diff
@@ -0,0 +1,100 @@
+# Certora Solidity Mutation Tester
+
+This is a mutation tester which
+ checks that variants of the original
+ solidity program do not pass the specification.
+If a mutated program passes the specification,
+it may indicate that the specification is vacuous or not rigorous enough.
+
+## Running the Mutation Tester
+
+- Example:
+```
+java -ea -jar $CERTORA/certora_jars/MutationTest.jar /path/to/config/file/Example.conf
+```
+
+- Remember to `yarn install` or `npm install` when working
+  on customer code if required by the customer code infra!
+
+- **NOTE: if a path has spaces, put quotes around it to ensure correct parsing by Kotlin's libraries**
+
+- Gambit **does** support `--staging`!
+**However, Gambit currently has trouble with
+`--send_only` and `--cloud` in the run scripts.
+If you have these flags, please remove them for now.**
+Apologies for the temporary inconvenience!
+
+The tool expects a configuration file (extension `.conf` is required)
+which is a JSON object.
+Here is an example configuration file:
+
+```
+{
+    "project_folder" : "Test/10Power",
+    "run_script" : "Test/10Power/runDefault.sh",
+    "gambit" : {
+        "filename" : "Test/10Power/TenPower.sol",
+        "solc" : "solc8.10"
+    },
+    "staging" : "master",
+    "manual_mutations" : {
+        "path/to/original/file.sol" : "path/to/manual/mutations/",
+        ...
+    },
+    ...
+}
+```
+
+Required Keys for the JSON Configuration File:
+- `"project_directory"` : the directory containing the original CVT project on which to perform mutation testing
+- `"run_script"` : the bash script used to run verification on the original project, usually `project_directory/run.sh` or similar
+- `"gambit"` : the JSON configuration element for invoking Gambit. May be a path to a gambit configuration file
+or the explicit JSON element contained therein. Refer to the open source Gambit repository for more details regarding Gambit configuration files.
+
+Optional Keys for the JSON Configuration File:
```

Comment:
move the internal Certora ones to the end, and maybe mark them as "for debugging"
--staging
--"use_cli_certora_run"`

Reply:
Done. A quick note about this though is that `--staging` is not default. By default it runs locally (because if you don't have CERTORAKEY, that can be a problem). Just FYI. We can change it if you think it's the right default but I would think it should be set by user.

Reply:
For any user other than a Certora dev, we shouldn't expect them to have a local installation.  The documentation and standard usage should assume that users are running on the cloud with a `CERTORAKEY`, and we should only mention running locally in internal documentation (on confluence).